pkg/odo/cli/catalog/describe/operator_backend.go

```diff
@@ -0,0 +1,89 @@
+package describe
+
+import (
+	"fmt"
+
+	"github.com/openshift/odo/pkg/log"
+	"github.com/openshift/odo/pkg/machineoutput"
+	"github.com/openshift/odo/pkg/service"
+	svc "github.com/openshift/odo/pkg/service"
+	olm "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+type operatorBackend struct {
+}
+
+func NewOperatorBackend() *operatorBackend {
+	return &operatorBackend{}
+}
+
+func (ohb *operatorBackend) CompleteDescribeService(dso *DescribeServiceOptions, args []string) error {
+	oprType, CR, err := service.SplitServiceKindName(args[0])
+	if err != nil {
+		return err
+	}
+	// we check if the cluster supports ClusterServiceVersion or not.
+	isCSVSupported, err := svc.IsCSVSupported()
+	if err != nil {
+		// if there is an error checking it, we return the error.
+		return err
+	}
+	// if its not supported then we return an error
+	if !isCSVSupported {
+		return errors.New("it seems the cluster doesn't support Operators. Please install OLM and try again")
```

Comment:
Nitpick.

I see `fmt.Errorf` in `ValidateDescribeService` while this one uses `errors.New`. I don't have a preference between the two. But could we try to use the same one?

Reply:
done